### PR TITLE
pighead: new, 0+git20201028

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

`pighead NULL => 0+git20201028`

Package(s) Affected
-------------------

pighead

Security Update?
----------------

No

Architectural Progress
----------------------

    

- [x] Architecture-independent `noarch`

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [ ] Architecture-independent `noarch`

### CODE REVIEW

**Comment by OriginCode** [comment]
LFGTM! as always
image

File: extra-misc/pighead/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,2 @@
+abinfo "Installing Pighead (Gumblex)..."
+install -Dvm755 "$SRCDIR"/pighead "$PKGDIR"/usr/bin/pighead
```
Comment (cthbleachbit) [inline extra-misc/pighead/autobuild/build:2]: 叫陈老板给他的猪头脚本上个 License

**Review by cthbleachbit — CHANGES_REQUESTED** [review]

File: extra-misc/pighead/autobuild/build (hunk 1 above)
Comment (OriginCode) [reply extra-misc/pighead/autobuild/build:2]: @chenx97 ~~建议 WTFPL~~

Comment (MingcongBai) [reply extra-misc/pighead/autobuild/build:2]: If he doesn't intend on including it, then we will ship it as such. This is not the place for memes.

**Review by MingcongBai — APPROVED** [review]

File: extra-misc/pighead/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=pighead
+PKGSEC=misc
+PKGDEP=""
+BUILDDEP=""
+PKGDES="Pighead of Gumblex, the logo of AOSC 7"
```
Comment (MingcongBai) [inline extra-misc/pighead/autobuild/defines]: AOSC OS Core 7 *.

Comment (MingcongBai) [reply extra-misc/pighead/autobuild/defines]: Or better yet, "A standalone version of the AOSC OS Core 7 logo - a "Gumblex" pig's head".

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Review by MingcongBai — APPROVED** [review]

**Review by cthbleachbit — APPROVED** [review]

File: extra-misc/pighead/autobuild/defines (hunk 2 above)
Comment (OriginCode) [reply extra-misc/pighead/autobuild/defines]: https://github.com/AOSC-Dev/aosc-os-abbs/pull/2438/commits/01925fcaee22b844cd034497e3d65d12dbcbf28d

File: extra-misc/pighead/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=pighead
+PKGSEC=misc
+PKGDEP=""
```
Comment (liushuyu) [inline extra-misc/pighead/autobuild/defines]:
```suggestion
PKGDEP="bash"
```

**Comment by MingcongBai** [comment]
@liushuyu Noted that Bash should be one of the PKGDEPs. Reverted status to "Changes Requested".

**Review by liushuyu — CHANGES_REQUESTED** [review]

**Review by liushuyu — APPROVED** [review]

**Pull request merged**